libsolidity/analysis/SyntaxChecker.cpp

```diff
@@ -141,7 +141,7 @@ bool SyntaxChecker::visit(ModifierDefinition const&)
 
 void SyntaxChecker::endVisit(ModifierDefinition const& _modifier)
 {
-	if (!m_placeholderFound)
+	if (_modifier.isImplemented() && !m_placeholderFound)
```

Comment:
Need to think if `_modifier.body().accept(*this)` is needed, just like `_function.body().accept(*this)`